steward/src/commands.rs

```diff
@@ -78,13 +78,21 @@ impl Configurable<StewardConfig> for EntryPoint {
             .map(PathBuf::from)
             .unwrap_or_else(|| CONFIG_FILE.into());
 
-        if filename.exists() {
+        let config_env_variable = std::env::var("VARIABLE").is_ok();
+
+        if config_env_variable {
+            let new_filename = self
+                .config
+                .as_ref()
+                .map(PathBuf::from)
+                .unwrap_or_else(|| config_env_variable.to_string().into());
```

Comment:
In this case we shouldn't try to read `self.config` again, if we're going to use the environment variable we should just construct a PathBuf for it directly.